backend/src/appointment/database/repo/external_connection.py

```diff
@@ -23,6 +23,9 @@ def create(db: Session, external_connection: ExternalConnection):
 def update_token(
     db: Session, token: str, subscriber_id: int, type: models.ExternalConnectionType, type_id: str | None = None
 ):
+    if type == models.ExternalConnectionType.google and type_id is None:
+        raise ValueError("Google external connection requires type_id to be provided")
```

Comment:
Is this propagated to the frontend? Then we might want to provide i10n for this. If this is only for us/Sentry, this is fine.